### ISSUE TITLE
Add an option to display the branch but not the status information.

### ISSUE THREAD

**Issue by pjvandehaar** [nojhan/liquidprompt#399]
Could you add an option to show the current branch without any status information? This is simply for (eg network) drives with slow IO and big git repos.

Like this:

``` bash
_lp_git_branch_color()
{
    local branch="$(_lp_git_branch)"
    [[ -n "${branch}" ]] && echo -ne "${LP_COLOR_UP}${branch}$NO_COL"
}
```

Alternatively, could this be displayed under paths in `LP_DISABLED_VCS_PATH`?

**Comment by dolmen** [comment]
Note that you already have the solution for yourself: in your `~/.bashrc`/`zshrc` you can redefine the `_lp_git_branch_color` function as you suggest just after loading LiquidPrompt.

**Comment by Rycieos** [comment]
Why would this be a better solution than using `LP_DISABLED_VCS_PATH`? The proposed option would disable it everywhere, where the vcs path would allow turning it off for only the slow network directories.

I would instead propose making `LP_DISABLED_VCS_PATH` disable all status information while keeping branch name on.

**Comment by Rycieos** [comment]
I just merged #635, which adds a separate theme `alternate_vcs` that only modifies the default theme's VCS display. Check out the implementation of that new theme in `themes/alternate_vcs/` as an example. The new theming engine in v2 makes it much simpler than before to make the modification you want to only display a branch name. You could probably make a theme that includes only this:

```bash
_lp_my_theme_theme_prompt() {
    _lp_default_theme_prompt_data

    if _lp_find_vcs && _lp_vcs_branch; then
        LP_VCS=" $lp_vcs_branch"
    else
        LP_VCS=
    fi

    _lp_default_theme_prompt_template
}
```
Of course [`_lp_vcs_branch()`](https://liquidprompt.readthedocs.io/en/v2.0.0-beta.2/functions/data/vcs.html#lp_vcs_branch) could return nothing if no branch is checked out, so I would recommend looking at how the default theme does branch detection (in `_lp_vcs_details_color()`) and maybe copy that.

As for

> I would instead propose making LP_DISABLED_VCS_PATH disable all status information while keeping branch name on.

I did not implement this. It would need another feature toggle, and would make more work for theme creators. I also think that older VCSs like SVN and Bazaar would still be too slow in large repos to just print a branch name. And like I mentioned above, I think you would need multiple calls still in case there was no current branch. I'm still open to the feature, just need to see more interest and discussion on it before I would implement it.
